### PR TITLE
Create(SelectLocationComponent)

### LINKED ISSUE #21: Feature#21-SelectLocationComponent
# REQUIREMENTS
Implements a component to select the location
Must allow you to choose between predetermined options

# DESIGN

### PR DESCRIPTION
**Pull request by AmarilloVel**
# Description
Implements a component that allows the user to select the country to travel
Using  react-native-dropdown-select-list library
  
  ## Screenshots 
image
image

### CODE REVIEW

File: flight-reservation/components/SelectLocationComponent/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            boxStyles={styles.boxStyles} 
+            dropdownStyles={styles.optionsBox} inputStyles={styles.searchText} 
+            searchicon=""/>
+            <View style={styles.line}></View>
+        </View>
+    )
+
+}
+
+export default SelectLocationComponent
```
Comment (Astroyayo) [inline flight-reservation/components/SelectLocationComponent/index.js]: pongamele un extra line porfa

File: flight-reservation/components/SelectLocationComponent/styles.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    },
+    line:{
+        borderWidth: 0.5,
+        borderColor: 'grey',
+        width: '90%',
+        alignSelf: 'center'
+    }
+});
+
+export default styles;
```
Comment (Astroyayo) [inline flight-reservation/components/SelectLocationComponent/styles.js]: extra line pls

**Review by Astroyayo — CHANGES_REQUESTED** [review]
👀

File: flight-reservation/components/SelectLocationComponent/index.js (hunk 1 above)
Comment (AmarilloVel) [reply flight-reservation/components/SelectLocationComponent/index.js]: Done!

**Review by Astroyayo — APPROVED** [review]

**Review by wiltrouble — APPROVED** [review]
look good, 👌

**Pull request merged**